### PR TITLE
philosophy question

### PR DESCRIPTION
**Pull request by S1lveria**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: ukrainian/trueStory.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿Вам здалося, що Ви прокинулися, але Ви все ще й досі уві сні.
```
Comment (casey-collab) [inline ukrainian/trueStory.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline ukrainian/trueStory.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**